Documents/UserManual/userManualFrench.md

```diff
@@ -0,0 +1,265 @@
+<img src="./Img/BluetoothUnlock.png" width="200"/>
+
+Avec une carte NFC:
+
+- Faites toucher l'appareil NFC et le SportShield.
+
+A propos de l'app SportShield, reportez-vous à la [section dédiée](#app).
+
+<br>
+SportShield est maintenant déverrouillé. L'alarme ne sonnera pas et vous pouvez reprendre le dispositif.
+
+<br>
+
+<div style="page-break-after: always;"></div>
+
+# Alarme
+
+Quand SportShield est verrouillé, chaque choc détecté sonnera une alarme. <br>L'intensité de l'alarme dépend de l'intensité du choc: <br>
+
+- 3 sons légers lorsqu'un choc léger est détecté.
+  
+- 5 sons longs et aigus lorsqu'un choc important est détecté.
+<br><br>
+
+De plus, vous pouvez arrêter l'alarme quand vous le souhaitez. Pour ce faire, vous pouvez utiliser le Bluetooth ou l'application. <br>
+
+Par Bluetooth: ce sont les mêmes étapes que le [verrouillage du SportShield](#how-to-unlock-sportshield).
+
+Avec une carte NFC, ce sont les mêmes étapes que le [déverrouillage du SportShield](#how-to-unlock-sportshield).
+
+A propos de l'application SportShield, reportez-vous à la [partie dédiée](#app).
+
+Quand l'alarme sonne, une notification est envoyé au smartphone pairé.<br>
+Pour plus d'informations sur les notifications, reportez-vous à la [section dédiée](#notifications).
+<br>
+
+<div style="page-break-after: always;"></div>
+
+# App
+
+L'application SportShield serait disponible sur l'Apple App Store et le Google Play Store.<br>
+Même si l'application n'a pas encore été synchronisée avec l'appareil SportShield. <br><br>Dans ce manuel, vous ne trouverez pas pour l'instant d'informations utiles sur cette application.
+
+<br><br>
+
+# Notifications
+
+Aucune application n'ayant été synchronisée avec le dispositif SportShield, la simulation mise en œuvre reproduit parfaitement son fonctionnement. <br>
+
+Les notifications contiennent des informations sur l'appareil, telles que les coordonnées GPS (longitude latitude) et le pourcentage de batterie restant.<br>
+
+<img src="Img/notificationFrench.png" width="400">
+
+<div style="page-break-after: always;"></div>
+
+# General Infos and Spécifications techniques
+
+## Composants
+
+Voici une liste des composants d'un appareil SportShield :
+
+- Xiao BLE Sense nrf52840
+- GNSS PA1010D
+- GSM/2G SIM800L Module
+- Electromagnet
+- Piezoelectric Buzzer
+- Lithium-Polymer Battery
+- NFC Antenna
+
+## Garantie de robustesse
+
+Le SportShield étant un dispositif antivol, il doit être suffisamment résistant et répondre à certains critères de résistance.
+
+**Étanchéité** : Le SportShield doit être étanche à l'eau et répondre aux indices de protection IP65, 66 ou 67.
+
+**Résistance au froid** : SportShield doit être résistant au froid et répondre aux critères TCR de 0 à 50 ppm/°C.
+
+**Résistance à la chaleur** : SportShield doit être résistant à la chaleur et répondre à des valeurs TCR comprises entre 100 et 500 ppm/°C.
+
+**Robustesse** : SportShield doit être suffisamment résistant aux tentatives d'effraction.
```

Comment:
"il doit être suffisamment résistant", as explained in English, it doesn't have to be enough, it's not the customer who makes the product. Replace that with : "Il est censé être suffisamment résistant".

Same for other lines.